__tests__/api tests/app.test.js

```diff
@@ -534,3 +535,14 @@ describe("DELETE: /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("200: Should return a status of 200 and a JSON object when the /api endpoint is called", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(getJSONfile);
```

Comment:
just a small gripe but i would see getJSONfile and think this was a function because of the verb 'get'. I would just call this something like endpoints, to make it clear